presto-tests/src/test/java/com/facebook/presto/tests/TestSqlFunctions.java

```diff
@@ -112,6 +112,42 @@ public void testCreateFunction()
         assertQueryFails("CREATE FUNCTION testing.test.foo(x smallint) RETURNS bigint LANGUAGE JAVA EXTERNAL NAME 'foo.from.another.library'", ".*mismatched input ''foo.from.another.library''. Expecting: <identifier>");
         assertQueryFails("CREATE FUNCTION testing.test.foo(x varchar) RETURNS varchar LANGUAGE JAVA EXTERNAL NAME", ".*mismatched input '<EOF>'. Expecting: <identifier>");
         assertQueryFails("CREATE FUNCTION testing.test.foo(x varchar) RETURNS varchar LANGUAGE UNSUPPORTED EXTERNAL", "Catalog testing does not support functions implemented in language UNSUPPORTED");
+
+        assertQuerySucceeds("CREATE FUNCTION testing.test.returnDouble() RETURNS DOUBLE RETURN 1");
+        String createFunctionReturnDoubleFormatted = "CREATE FUNCTION testing.test.\"returnDouble\" ()\n" +
```

Comment:
nit: Let use lower-underscore format for function names (as a convention) in the test. e.g. testing.test.return_double. Same and below.

Reply:
Let's also add some assertion to this case trying to run this function
```
computeActual("SELECT testing.test.return_double() + 1")
assert ...
```

Reply:
fixed.